atom/browser/lib/bluetooth_chooser.cc

```diff
@@ -69,8 +68,9 @@ void BluetoothChooser::ShowDiscoveryState(DiscoveryState state) {
       break;
     case DiscoveryState::IDLE:
       if (device_list_.empty()) {
-        auto event = ++num_retries_ > kMaxScanRetries ? Event::CANCELLED
-                                                      : Event::RESCAN;
+        auto event = ++num_retries_ > kMaxScanRetries ?
```

Comment:
The original code does not violate coding style and actually matches the common practice in Chromium's codebase.